src/gui/folderwatcher.cpp

```diff
@@ -190,4 +206,62 @@ void FolderWatcher::changeDetected(const QStringList &paths)
     }
 }
 
+void FolderWatcher::possiblyAddUnlockedOfficeFile(const QString &path, QSet<QString> &container)
```

Comment:
I am not a fan of using non const reference for arguments
cannot you just a return a new `QSet` that you insert into `unlockedOfficeFiles`
not sure that would change the performance in a noticeable way but would make it easier to read the code